spark/src/main/scala/io/delta/sharing/spark/DeltaSharingOptions.scala

```diff
@@ -79,14 +82,31 @@ trait DeltaSharingReadOptions extends DeltaSharingOptionParser {
     }
   }
 
-  val timestampAsOf = options.get(TIME_TRAVEL_TIMESTAMP)
+  val timestampAsOf = options.get(TIME_TRAVEL_TIMESTAMP).map(getFormattedTimestamp(_))
 
   def isTimeTravel: Boolean = versionAsOf.isDefined || timestampAsOf.isDefined
 
+  // Parse the input timestamp string and TimestampType, and generate a formatted timestamp string
+  // in the ISO8601 format, in the UTC timezone, such as 2022-01-01T00:00:00Z.
```

Comment:
Does it assume that the input time is in UTC?
Consider adding a comment on the timezones are handled.

Reply:
updated the comment. In the list of examples, there's one indicating it can be from any timezone.